Sources/TuistKit/Utils/ResourceLocator.swift

```diff
@@ -48,7 +48,12 @@ final class ResourceLocator: ResourceLocating {
     private func frameworkPath(_ name: String) throws -> AbsolutePath {
         let frameworkNames = ["\(name).framework", "lib\(name).dylib"]
         let bundlePath = AbsolutePath(Bundle(for: GraphManifestLoader.self).bundleURL.path)
-        let paths = [bundlePath, bundlePath.parentDirectory]
+        let paths = [
+            AbsolutePath("/Users/marekfort/Development/tuist/.build/x86_64-apple-macosx"),
```

Comment:
If users had a `marekfort` in their environments :trollface:. We need to remove this one @fortmarek

Reply:
Jeez 🤦‍♂ I had some problems with running from Xcode and forgot to delete this.